frontend/app/src/components/dialogs/WatcherDialog.vue

```diff
@@ -310,7 +312,7 @@ const cancel = () => {
               />
             </v-col>
             <v-col class="d-flex align-center justify-space-between" cols="2">
-              <v-btn icon @click="editWatcher(loadedWatchers[key])">
+              <v-btn icon @click="() => editWatcher(loadedWatchers[key])">
```

Comment:
is there any reason we can't do `editWatcher(key)`

Reply:
yeah we can, i was experimenting and forgot to revert this, too sleepy

Reply:
> yeah we can, i was experimenting and forgot to revert this, too sleepy

Do it in another PR then it's fine.